mobile/src/TopicsModal.js

```diff
@@ -154,10 +154,11 @@ export default class TopicsModal extends Component {
               <FilterCell
                 item={item}
                 key={i}
-                select={false}
+                select={this.props.selectedFilters.find(filter => filter === item)}
```

Comment:
You can use `includes` instead of `find` in this case. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes